docs/modules/integrations/pages/zephyr-exporter.adoc

```diff
@@ -24,6 +26,10 @@ Features:
 |false
 |Property for update existing executions statuses only.
 
+|`zephyr.exporter.jira-instance-key`
+|false
+|The key of previousle configured JIRA instance, in case of missing value it will be evaluated automatically based on issue keys being exported
```

Comment:
```suggestion
|The key of the configured JIRA instance, in case of missing value it will be evaluated automatically based on issue keys being exported
```